src/components/AddRepsModal/LiftInfo/Links.tsx

```diff
@@ -0,0 +1,93 @@
+import Octicon, { getIconByName } from "@githubprimer/octicons-react";
+import * as React from "react";
+import AnimateHeight from "react-animate-height";
+import {
+  Button,
+  FormText,
+  Input,
+  InputGroup,
+  InputGroupAddon
+} from "reactstrap";
+import { LiftInfoLink } from "../../../types/LiftTypes";
+
+type Prop = {
+  onLinkAdd: () => void;
+  onLinkRemove: (index: number) => void;
+  onLinkChange: (
+    event: React.ChangeEvent<HTMLInputElement>,
+    index: number
+  ) => void;
+  links: LiftInfoLink[];
+};
+
+const LINK_ELEMENT_HEIGHT = 40;
+const MAX_LINKS_NUMBER = 3;
+
+const Links = (props: Prop) => {
+  const { links, onLinkChange, onLinkAdd, onLinkRemove } = props;
+  const linksContainerHeight = links.length * LINK_ELEMENT_HEIGHT;
+  const canAddMoreLinks = links.length < MAX_LINKS_NUMBER;
+
+  return (
+    <div className="mt-2">
+      <AnimateHeight
+        duration={350}
+        className="pt-1 mx--1"
```

Comment:
Should it be `mx-1` or `mx--1`?

Reply:
there is already a bootstrap class `mx-1` which means horizontal padding of 1 bootstrap unit (0.25 rem to be exact) but `mx--1` is my own class which sets negative margin of -0.25 rem. Is the name of the class confusing?

Reply:
It is a bit confusing, since it wasn't obvious that the second `-` was a `minus` sign, but it makes sense now that I know.